Source/Core/VideoBackends/OGL/FramebufferManager.cpp

```diff
@@ -76,173 +135,81 @@ FramebufferManager::FramebufferManager(int targetWidth, int targetHeight, int ms
 
   glActiveTexture(GL_TEXTURE9);
 
-  GLuint glObj[3];
-  glGenTextures(3, glObj);
-  m_efbColor = glObj[0];
-  m_efbDepth = glObj[1];
-  m_efbColorSwap = glObj[2];
-
   m_EFBLayers = (g_ActiveConfig.iStereoMode > 0) ? 2 : 1;
   m_efbFramebuffer.resize(m_EFBLayers);
   m_resolvedFramebuffer.resize(m_EFBLayers);
 
-  // OpenGL MSAA textures are a different kind of texture type and must be allocated
-  // with a different function, so we create them separately.
+  GLenum depth_internal_format = GL_DEPTH_COMPONENT32F;
```

Comment:
I can extract it to a separate function if you want. One thing I really like about keeping it inline is that all uses of m_enable_stencil buffer are in this same chunk of 80 lines or so, and you get the benefit of seeing which values go to which variables (since static-typing is useless here...). Do you want me to make it a separate function?